src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs

```diff
@@ -26,7 +26,8 @@ internal sealed class MemberExpansion : Expansion
             DkmClrValue value,
             ExpansionFlags flags,
             Predicate<MemberInfo> predicate,
-            ResultProvider resultProvider)
+            ResultProvider resultProvider,
+            bool isProxyType)
```

Comment:
Committed a test to ensure that the behaviour on non-proxy types is not affected by this PR in 97164c9.